### PR TITLE
Ron/special commands rename

### PR DESCRIPTION
**Pull request by MrMeemus**
Renaming the special commands to be intuitive to their description.
Updated usage guide.
Update tests.

### CODE REVIEW

**Comment by abhisheksinha89** [comment]
> mssql-cli>\n dept Human

Incorrect, based on the table below it should be \ln

---
Refers to: doc/usage_guide.md:206 in 749a380. [](commit_id = 749a38052fed3bef799919ba691bc32a7a0ade67, deletion_comment = False)

**Comment by abhisheksinha89** [comment]
> mssql-cli>\n dept

\ln?

---
Refers to: doc/usage_guide.md:197 in 749a380. [](commit_id = 749a38052fed3bef799919ba691bc32a7a0ade67, deletion_comment = False)

File: doc/usage_guide.md [hunk 1]
```diff
@@ -211,17 +211,18 @@ Below table summarizes the special commands supported
...
-| \\? | \\? | Show this help.
+|\d   | \d OBJECT                        | List or describe database objects. Calls sp_help.
+|\dn  | \dn [name]                       | Delete a named query.                            
+|\e   | \e [file]                        | Edit the query with external editor.             
+|\ld  | \ld[+] [pattern]                 | List databases.                                  
+|\lf  | \lf[+] [pattern]                 | List functions.                                  
+|\li  | \li[+] [pattern]                 | List indexes.                                    
+|\ll  | \ll[+] [pattern]                 | List logins and associated roles.                
+|\ln  | \ln[+] [name] [param1 param2 ...]| List or execute named queries.                   
+|\ls  | \ls[+] [pattern]                 | List schemas.
```
Comment (pensivebrian) [inline doc/usage_guide.md]: should be \n

File: mssqlcli/packages/special/commands.py [hunk 2]
```diff
@@ -180,7 +180,7 @@ def list_logins(mssqlcliclient, pattern, verbose):
     return mssqlcliclient.execute_multi_statement_single_batch(base_query)
 
 
-@special_command('\\n', '\\n[+] [name] [param1 param2 ...]', 'List or execute named queries.')
+@special_command('\\ln', '\\ln[+] [name] [param1 param2 ...]', 'List or execute named queries.')
 def execute_named_query(mssqlcliclient, pattern, **__):
```
Comment (pensivebrian) [inline mssqlcli/packages/special/commands.py]: \n

File: tests/test_special.py [hunk 3]
```diff
@@ -104,28 +104,28 @@ def test_named_queries_commands(self):
...
                 self.assertTrue(len(rows) == 1)
                 self.assertTrue(len(col) == 1)
 
             # Delete a named query that was created
-            list(client.execute_multi_statement_single_batch('\\nd test123'))
+            list(client.execute_multi_statement_single_batch('\\dn test123'))
 
             # Number of named queries should have reduced by 1
-            for rows, col, message, sql, is_error in client.execute_multi_statement_single_batch('\\n'):
+            for rows, col, message, sql, is_error in client.execute_multi_statement_single_batch('\\ln'):
```
Comment (abhisheksinha89) [inline tests/test_special.py]: >\\ln [](start = 98, length = 4)

\n just make sure

File: mssqlcli/packages/special/namedqueries.py [hunk 4]
```diff
@@ -8,10 +8,10 @@ class NamedQueries(object):
 Examples:
 
     # Save a new named query.
-    > \\ns simple select * from abc where a is not Null;
+    > \\sn simple select * from abc where a is not Null;
 
     # List all named queries.
-    > \\n
+    > \\ln
```
Comment (abhisheksinha89) [inline mssqlcli/packages/special/namedqueries.py]: >\\ln [](start = 6, length = 4)

\n

**Pull request merged**